CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Removed
 - Ability to create new components when moving files.
```

Comment:
It is weird, this isn't here in develop or next-interfaces, not sure why it isn't updating here properly.